### PR TITLE
Capture acceptance test results in an xunit.xml file

### PR DESCRIPTION
**Pull request by benpatterson**
This will allow test results to be more easily consumed by:
* CI infrastructure
* Developers who want to understand results
* Logging infrastructure we use wherein we can roll together test results across repos

### CODE REVIEW

**Comment by benpatterson** [comment]
@schenedx @jimabramson @rlucioni 

PTAL

File: .gitignore [hunk 1]
```diff
@@ -38,6 +38,7 @@ htmlcov
 .tox
 nosetests.xml
 unittests.xml
+acceptance_tests/xunit.xml
```
Comment (rlucioni) [inline .gitignore]: Would you mind doing this for ecommerce, too? This wasn't done in https://github.com/edx/ecommerce/pull/665.

**Comment by rlucioni** [comment]
👍

File: .gitignore (hunk 1 above)
Comment (benpatterson) [inline .gitignore]: Don't mind at all...occurred to me afterwards. PR coming...

**Comment by schenedx** [comment]
:+1:

**Pull request merged**